xchange-upbit/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.knowm.xchange</groupId>
+		<artifactId>xchange-parent</artifactId>
+		<version>4.3.8-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>xchange-upbit</artifactId>
+
+	<name>XChange Upbit</name>
+	<description>XChange implementation for Upbit</description>
+
+	<url>http://knowm.org/open-source/xchange/</url>
+	<inceptionYear>2012</inceptionYear>
+
+	<organization>
+		<name>Knowm Inc.</name>
+		<url>http://knowm.org/open-source/xchange/</url>
+	</organization>
+
+	<!-- Parent provides default configuration for dependencies -->
+	<dependencies>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>xchange-core</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.auth0</groupId>
+			<artifactId>java-jwt</artifactId>
```

Comment:
I'd rather not have to introduce another dependency to XChnage. Can you use one that's already included in the parent pom?

Reply:
Ah. Yes, I edited